components/octopost/Works/Card.tsx

```diff
@@ -0,0 +1,66 @@
+'use client';
+
+import { motion } from "framer-motion";
+import React, { useState } from 'react';
+import ColorfulEllipse from "./assets/ ColorfulEllipse";
+import CloudIcon from "./assets/CloudIcon";
+import Ellipse from "./assets/Ellipse";
+import ColorfulCloud from "./assets/ColorfulCloud";
+
+type CardProps = {
+  title: string,
+  text: string,
+}
+
+function Card({
+  title,
+  text,
+}: CardProps) {
+  const [isHovered, setIsHovered] = useState(false);
+
+  
+  return (
+    <motion.div className="w-[265px] bg-white shadow-dropshadow-card h-96 rounded-xl px-8 pt-16 pb-10 flex flex-col justify-between items-center relative overflow-hidden group"
+    whileHover={{scale: 1.02}}
+    transition={{duration: 0.5}}
+    onMouseEnter={() => setIsHovered(true)}
+    onMouseLeave={() => setIsHovered(false)}>
+      <div className="absolute top-0 left-0">

```

Comment:
Não precisa alterar, aqui quando for usar top-0 left-0 da pra usar inset-0.

```suggestion
      <div className="absolute top-0 left-0">
      <div className="absolute inset-0">
```